server/service/src/purchase_order_line/insert/validate.rs

```diff
@@ -40,11 +40,15 @@ pub fn validate(
         return Err(InsertPurchaseOrderLineError::CannotEditPurchaseOrder);
     }
 
-    let item = match ItemRowRepository::new(connection).find_one_by_id(&input.item_id)? {
-        Some(item) => item,
-        None => {
-            return Err(InsertPurchaseOrderLineError::ItemDoesNotExist);
-        }
+    let item_repo = ItemRowRepository::new(connection);
```

Comment:
This is temp... it'll all be moved to upsert soon so would be checking enum and only pass in on input for item_id_or_code

Reply:
Sounds good 👍